experimenter/experimenter/klaatu/client.py

```diff
@@ -81,7 +81,7 @@ def run_test(
             "ref": "main",
             "inputs": {
                 "slug": experiment_slug,
-                "branch": json.dumps([branch_slug]),
+                "branch": json.dumps([branch_slugs]),
```

Comment:
```suggestion
                "branch": json.dumps(branch_slugs),
```
seems like this shouldn't become double nested?

Reply:
Ah yes, good catch.